addons/common/stringtable.xml

```diff
@@ -999,94 +999,94 @@
-            <German>ACE3 Bewegung</German>
-            <Polish>ACE3 Ruch</Polish>
-            <Portuguese>Movimento ACE3</Portuguese>
-            <Russian>ACE3 Перемещение</Russian>
-            <Spanish>ACE3 Movimiento</Spanish>
-            <Czech>ACE3 Pohyb</Czech>
-            <Italian>Movimento ACE3</Italian>
-            <French>ACE3 Mouvement</French>
-            <Japanese>ACE3 移動</Japanese>
-            <Korean>ACE3 움직임</Korean>
-            <Chinese>ACE3 動作按鍵</Chinese>
-            <Chinesesimp>ACE3 动作按键</Chinesesimp>
+            <English>ACE Movement</English>
+            <German>ACE Bewegung</German>
+            <Polish>ACE Ruch</Polish>
+            <Portuguese>ACE Movimento</Portuguese>
+            <Russian>ACE Перемещение</Russian>
+            <Spanish>ACE Movimiento</Spanish>
+            <Czech>ACE Pohyb</Czech>
+            <Italian>ACE Movimento</Italian>
+            <French>ACE Mouvement</French>
+            <Japanese>ACE 移動</Japanese>
+            <Korean>ACE 움직임</Korean>
+            <Chinese>ACE 動作按鍵</Chinese>
+            <Chinesesimp>ACE 动作按键</Chinesesimp>
         </Key>
         <Key ID="STR_ACE_Common_ACEKeybindCategoryScopeAdjustment">
-            <English>ACE3 Scope Adjustment</English>
-            <German>ACE3 Visiereinstellung</German>
-            <Polish>ACE3 Regulacja optyki</Polish>
-            <Portuguese>Ajuste de luneta ACE3</Portuguese>
-            <Russian>ACE3 Прицелы</Russian>
-            <Czech>ACE3 Nastavení optiky</Czech>
-            <Spanish>ACE3 Ajuste de miras</Spanish>
-            <Italian>Regolazione Ottiche ACE3</Italian>
-            <French>ACE3 Ajustement de la lunette </French>
-            <Japanese>ACE3 スコープ調節</Japanese>
-            <Korean>ACE3 조준경 조정</Korean>
-            <Chinese>ACE3 瞄準鏡調節按鍵</Chinese>
-            <Chinesesimp>ACE3 瞄准镜调节按键</Chinesesimp>
+            <English>ACE Scope Adjustment</English>
+            <German>ACE Visiereinstellung</German>
+            <Polish>ACE Regulacja optyki</Polish>
+            <Portuguese>ACE Ajuste de luneta</Portuguese>
+            <Russian>ACE Прицелы</Russian>
+            <Czech>ACE Nastavení optiky</Czech>
+            <Spanish>ACE Ajuste de miras</Spanish>
+            <Italian>ACE Regolazione Ottiche</Italian>
+            <French>ACE Ajustement de la lunette </French>
+            <Japanese>ACE スコープ調節</Japanese>
+            <Korean>ACE 조준경 조정</Korean>
+            <Chinese>ACE 瞄準鏡調節按鍵</Chinese>
+            <Chinesesimp>ACE 瞄准镜调节按键</Chinesesimp>
         </Key>
         <Key ID="STR_ACE_Common_ACEKeybindCategoryVehicles">
-            <English>ACE3 Vehicles</English>
-            <German>ACE3 Fahrzeuge</German>
-            <Polish>ACE3 Pojazdy</Polish>
-            <Portuguese>Veículos ACE3</Portuguese>
-            <Russian>ACE3 Транспорт</Russian>
-            <Czech>ACE3 Vozidla</Czech>
-            <Spanish>ACE3 Vehículos</Spanish>
-            <Italian>Veicoli ACE3</Italian>
```

Comment:
My only concern is whether this is no longer localised correctly?

Reply:
Think of it as: `<mod> <category>`.

Reply:
It's combining 2 nouns, I don't think the order could be that bad. 
("Equipment ACE3" sounds a little weird, but can be understood just fine)
Having the prefix first will be a big help for categories as they get sorted alphabetically and I think we want all of ace together.